modules/src/ics02_client/client_def.rs

```diff
@@ -119,11 +122,14 @@ pub enum AnyClientState {
     Mock(MockClientState),
 }
 
-impl AnyClientState {
-    pub fn from_any(any: prost_types::Any) -> Result<Self, Error> {
-        match any.type_url.as_str() {
+impl TryFromRaw for AnyClientState {
+    type RawType = prost_types::Any;
+    type Error = Error;
+
+    fn try_from(value: Self::RawType) -> Result<Self, Self::Error> {
+        match value.type_url.as_str() {
             "ibc.tendermint.ClientState" => {
```

Comment:
Could you change all `type_url` strings to start with `/`?
```suggestion
            "/ibc.tendermint.ClientState" => {
```
Made that change and tried the client state query on latest gaia chains with clients created by the Go relayer and I get this which is nice:
```
rrly -c simple_config.toml query client state ibc0 ibconeclient -p false
    Finished dev [unoptimized + debuginfo] target(s) in 0.13s
     Running `target/debug/relayer -c simple_config.toml query client state ibc0 ibconeclient -p false`
     Options QueryClientStateOptions { client_id: ClientId("ibconeclient"), height: 0, proof: false }
client state query result:  Tendermint(ClientState { chain_id: "ibc1", trusting_period: 1209600s, unbonding_period: 1814400s, max_clock_drift: 60s, latest_height: block::Height(112), frozen_height: block::Height(0) })
```

For the consensus state I cannot get it to work even from curl or browser. But at least the client state looks good :)

Reply:
WOhoo!

> Could you change all type_url strings to start with /?

Yep, changed all of them.